src/soundio/soundmanagerconfig.cpp

```diff
@@ -51,14 +51,25 @@ SoundManagerConfig::SoundManagerConfig(SoundManager* pSoundManager)
       m_iNumMicInputs(0),
       m_bExternalRecordBroadcastConnected(false),
       m_pSoundManager(pSoundManager) {
-    m_configFile = QFileInfo(QDir(CmdlineArgs::Instance().getSettingsPath()).filePath(SOUNDMANAGERCONFIG_FILENAME));
+    CmdlineArgs& cla = CmdlineArgs::Instance();
+    QDir settDir = QDir(cla.getSettingsPath());
+    // Try to load custom sound config first
+    QString customCfg = cla.getSoundConfig();
+    if (!customCfg.isEmpty()) {
+        QFileInfo customCfgFileInfo = QFileInfo(settDir.filePath(customCfg));
+        QFile customCfgFile(customCfgFileInfo.absoluteFilePath());
```

Comment:
yeah I know, outcome seems to be the same and I already use it in the profile selector branch.
I just thought I overlooked the reason for creating QFile and using QFile::open, but appearantly that's because the file needs to be opened anyway.

Reply:
done.